package.json

```diff
@@ -1,57 +1,75 @@
 {
-	"name": "virtual-mmx",
-	"version": "0.1.0-alpha",
-	"private": true,
-	"dependencies": {
-		"vmmx-schema": "wintergatan-community/vmmx-schema",
-		"@testing-library/jest-dom": "^4.2.4",
-		"@testing-library/react": "^9.5.0",
-		"@testing-library/user-event": "^7.2.1",
-		"@types/jest": "^24.9.1",
-		"@types/node": "^12.12.42",
-		"@types/react": "^16.9.35",
-		"@types/react-dom": "^16.9.8",
-		"bootstrap": "^4.5.0",
-		"react": "^16.13.1",
-		"react-bootstrap": "^1.0.1",
-		"react-dom": "^16.13.1",
-		"react-scripts": "3.4.1",
-		"tone": "^13.8.25",
-		"typescript": "^3.7.5"
-	},
-	"scripts": {
-		"start": "react-scripts start",
-		"build": "react-scripts build",
-		"test": "react-scripts test --passWithNoTests",
-		"eject": "react-scripts eject"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged"
-		}
-	},
-	"lint-staged": {
-		"*.{js,ts,tsx}": "eslint --cache --fix --ignore-path .eslintignore",
-		"*.{js,ts,tsx,css,md}": "prettier --write --ignore-path .eslintignore"
-	},
-	"eslintConfig": {
-		"extends": "react-app"
-	},
-	"browserslist": {
-		"production": [
-			">0.2%",
-			"not dead",
-			"not op_mini all"
-		],
-		"development": [
-			"last 1 chrome version",
-			"last 1 firefox version",
-			"last 1 safari version"
-		]
-	},
-	"devDependencies": {
-		"husky": "^4.2.5",
-		"lint-staged": "^10.2.6",
-		"prettier": "^2.0.5"
-	}
+    "name": "virtual-mmx",
+    "version": "0.1.0-alpha",
+    "private": true,
+    "description": "Web application simulating Wintergatans Marble Maschine X",
```

Comment:
Typo with Maschine

Reply:
Fixed in 6e66da9d377e7dbd1846ab86eb016da26ef1b156